python-checks/src/main/java/org/sonar/python/checks/JwtVerificationCheck.java

```diff
@@ -87,6 +88,11 @@ private static void verifyCallExpression(SubscriptionContext ctx) {
       RegularArgument verifyArg = TreeUtils.argumentByKeyword("verify", call.arguments());
       if (verifyArg != null && Expressions.isFalsy(verifyArg.expression())) {
         ctx.addIssue(verifyArg, MESSAGE);
+      } else {
+        Optional.ofNullable(TreeUtils.argumentByKeyword("options", call.arguments()))
```

Comment:
Is this intentional that this will also work for `"jose.jws.verify"`? I don't see it mentioned explicitly in the ticket.
If yes, I think it would make sense to add a test for it. If not, probably we should exclude it?

Reply:
Good catch